### DISCUSSION TITLE
%E6%90%9C%E7%8B%90%E6%96%B0%E9%97%BB/news_%E5%BC%A0%E5%85%8B%E8%8E%8E%EF%BC%9A%E6%88%91%E6%98%AF%E5%8F%98%E6%80%A7%E4%BA%BA_-_%E6%96%B0%E9%97%BB/

### DISCUSSION THREAD

**Discussion by giscus** [project-polymorph/trans-digital-cn#9] [Announcements]
# %E6%90%9C%E7%8B%90%E6%96%B0%E9%97%BB/news_%E5%BC%A0%E5%85%8B%E8%8E%8E%EF%BC%9A%E6%88%91%E6%98%AF%E5%8F%98%E6%80%A7%E4%BA%BA_-_%E6%96%B0%E9%97%BB/

本存档库包含各类中文跨性别、多元性别、酷儿与双性人等的新闻网页存档

https://news.transchinese.org/%E6%90%9C%E7%8B%90%E6%96%B0%E9%97%BB/news_%E5%BC%A0%E5%85%8B%E8%8E%8E%EF%BC%9A%E6%88%91%E6%98%AF%E5%8F%98%E6%80%A7%E4%BA%BA_-_%E6%96%B0%E9%97%BB/

**Comment by yunwei37** [comment]
测试
